depthai_helpers/arg_manager.py

```diff
@@ -54,6 +54,10 @@ def parse_args():
                         help="Enable stereo 'Left-Right check' feature.")
     parser.add_argument("-scale", "--scale", default=1.0, type=float,
                         help="Scale factor for the output window. Default: %(default)s")
+    parser.add_argument("-maxd", "--max_depth", default=10000, type=int,
+                        help="Maximum expected depth distance, 1000 = 1m. Default: %(default)s")
```

Comment:
instead:
"Maximum depth distance for spatial coordinates?"

Reply:
And "Units are in mm."

Reply:
Agreed, just updated